content/sensu-go/6.2/operations/deploy-sensu/secure-sensu.md

```diff
@@ -33,20 +33,20 @@ To properly secure etcd communication, replace the default parameter values in y
 
 1. Replace the placeholder with the path to your certificate and key for the `etcd-cert-file` and `etcd-key-file` to secure client communication:
 {{< code yml >}}
-etcd-cert-file: "/path/to/your/cert"
-etcd-key-file: "/path/to/your/key"
+etcd-cert-file: "/etc/sensu/tls/agent.pem"
+etcd-key-file: "/etc/sensu/tls/agent-key.pem"
 {{< /code >}}
 
 2. Replace the placeholder with the path to your certificate and key for the `etcd-peer-cert-file` and `etcd-peer-key-file` to secure cluster communication:
 {{< code yml >}}
-etcd-peer-cert-file: "/path/to/your/peer/cert"
-etcd-peer-key-file: "/path/to/your/peer/key"
+etcd-peer-cert-file: "/etc/sensu/tls/backend-1.pem"
+etcd-peer-key-file: "/etc/sensu/tls/backend-1-key.pem"
 {{< /code >}}
 
-3. Replace the placeholder with the path to your certificate and key for the `etcd-trusted-ca-file` and `etcd-peer-trusted-ca-file` to secure communication with the etcd client server and between etcd cluster members:
+3. Replace the placeholder with the path to your `ca.pem` certificate and key for the `etcd-trusted-ca-file` and `etcd-peer-trusted-ca-file` to secure communication with the etcd client server and between etcd cluster members:
 {{< code yml >}}
-etcd-trusted-ca-file: "/path/to/your/ca/file"
-etcd-peer-trusted-ca-file: "/path/to/your/peer/ca/file"
+etcd-trusted-ca-file: "/etc/sensu/tls/ca.pem"
+etcd-peer-trusted-ca-file: "/etc/sensu/tls/ca.pem"
```

Comment:
Is it correct that *both* `etcd-trusted-ca-file` and `etcd-peer-trusted-ca-file` should have the same ca.pem as their values?